sdk/containerregistry/container-registry/package.json

```diff
@@ -36,16 +36,16 @@
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
     "generate:client": "autorest --typescript ./swagger/README.md",
-    "integration-test:browser": "dev-tool run test:browser",
+    "integration-test:browser": "echo skipped",
```

Comment:
Can we also skip the browser build target:

https://github.com/Azure/azure-sdk-for-js/blob/844fd05c956eceff445465d4a31b4dbdb09c1edb/sdk/web-pubsub/web-pubsub/package.json#L11

and use `--browser-test=false` with dev-tool to avoid the extra bundling overhead?